### DISCUSSION TITLE
Question: When avaliable the Charm's?

### DISCUSSION THREAD

**Discussion by KaV3R** [ianlucas/cs2-inventory-simulator#120] [General]
Hi^^ I just wanna ask, when is avaliable the charms for the skins? 

![image_2024-10-20_173346804](https://github.com/user-attachments/assets/486c8f13-236c-4d95-b23b-578ce5359032)

**Comment by ianlucas** [comment]
That's being tracked on #110. No ETA, but donations are always appreciated to speed up the process.
